.gitignore

```
@@ -413,5 +413,6 @@ temp/
 .whl
 .tar.gz
```

```suggestion
*.whl
*.tar.gz
```

I know it's unrelated, but could you update this?